src/System.Runtime.Loader/tests/DefaultContext/DefaultLoadContextTest.cs

```diff
@@ -32,100 +32,57 @@ protected override Assembly Load(AssemblyName assemblyName)
 
     public class OverrideDefaultLoadContext : AssemblyLoadContext
     {
-        private bool m_fLoadedFromContext = false;
+        public bool LoadedFromContext { get; set; } = false;
 
         protected override Assembly Load(AssemblyName assemblyName)
         {
             // Override the assembly that was loaded in DefaultContext.
             string assemblyPath = Path.Combine(Path.GetDirectoryName(typeof(string).Assembly.Location), assemblyName.Name + ".dll");
             Assembly assembly = LoadFromAssemblyPath(assemblyPath);
-            m_fLoadedFromContext = true;
+            //m_fLoadedFromContext = true;
+            LoadedFromContext = true;
             return assembly;
         }
-
-        public bool LoadedFromContext
-        {
-            get
-            {
-                return m_fLoadedFromContext;
-            }
-            set
-            {
-                m_fLoadedFromContext = value;
-            }
-        }
     }
 
     [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "AssemblyLoadContext not supported on .Net Native")]
     public class DefaultLoadContextTests
     {
-        private static string s_loadFromPath = null;
+        private const string TestAssemblyName = "System.Runtime.Loader.Noop.Assembly";
+        private string _assemblyPath;
 
         // Since the first non-Null returning callback should stop Resolving event processing,
         // this counter is used to assert the same.
-        private static int s_NumNonNullResolutions = 0;
+        private int _numNonNullResolutions = 0;
 
-        private static Assembly ResolveAssembly(AssemblyLoadContext sender, AssemblyName assembly)
+        public DefaultLoadContextTests()
         {
-            string assemblyFilename = assembly.Name + ".dll";
-            s_NumNonNullResolutions++;
-
-            return sender.LoadFromAssemblyPath(Path.Combine(s_loadFromPath, assemblyFilename));
+            _assemblyPath = Path.GetFullPath("System.Runtime.Loader.Noop.Assembly_test.dll");
```

Comment:
It is not good to be making assumptions about the current directory when the test is run.